src/tasks/deploy_multisig.js

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright: Ambrosus Inc.
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+import TaskBase from './base/task_base';
+import commandLineArgs from 'command-line-args';
+import {appendFile} from '../utils/file';
+import config from '../../config/config';
+import {multisig} from '../../src/contract_jsons';
+import {
+  APPROVALS_REQUIRED
+} from '../constants';
+
+export default class DeployMultisigTask extends TaskBase {
+  constructor(deployActions, multiplexerWrapper) {
+    super();
+    this.deployActions = deployActions;
+    this.multiplexerWrapper = multiplexerWrapper;
+  }
+
+  async execute(args) {
+    console.log('Deploying MultiSigWallet contract.');
+
+    const options = this.parseOptions(args);
+    if (options === null) {
+      return;
+    }
+
+    let approvalAdresses;
+    if (typeof config.multisigApprovalAddresses !== 'undefined') {
+      approvalAdresses = Array.from(config.multisigApprovalAddresses.split(','));
```

Comment:
Why do we need Array.from here?

Reply:
It was just convinient to see the type (array) at once. Nevertheless it works without Array.from.
Fixed